wdl-engine/src/stdlib/size.rs

```diff
@@ -40,8 +61,17 @@ fn size(context: CallContext<'_>) -> BoxFuture<'_, Result<Value, Diagnostic>> {
                 .coerce_argument(1, PrimitiveType::String)
                 .unwrap_string();
 
-            unit.parse()
-                .map_err(|_| invalid_storage_unit(&unit, context.arguments[1].span))?
+            unit.parse().map_err(|_| {
+                function_call_failed(
+                    "size",
+                    format!(
+                        "invalid storage unit `{unit}`: supported units are `B`, `KB`, `K`, `MB`, \
+                         `M`, `GB`, `G`, `TB`, `T`, `KiB`, `Ki`, `MiB`, `Mi`, `GiB`, `Gi`, `TiB`, \
+                         and `Ti`",
```

Comment:
should the supported units in this message be derived from something?

Reply:
At best I think we could have an iterator on `StorageUnit` that iterates over this list, but then we'd have to format it. As this message is just moved from the now-removed diagnostic, I don't think this is necessary to be honest.

Reply:
Ok. I'm just looking at a long hardcoded list and thinking "that could be annoying if the units ever change", but in practice that seems unlikely to happen. I'm fine with this as it is 👍